routes.js

```diff
@@ -0,0 +1,63 @@
+var restify = require('restify');
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+    var server = restify.createServer();
+    server.use(restify.bodyParser());
```

Comment:
This should be in server.js not here, you would't want to copy/paste this into every controller